website/learn/blockchain/genesis/index.md

```diff
@@ -40,8 +40,9 @@ These parameters are hardcoded into the project and include:
   are defined at genesis time and ensure that the entire network operates within the same configuration.
   These consensus parameters are discussed in detail in the consensus section of the documentation.
 - **Treasury Account**: The [treasury account]({{ site.baseurl }}/learn/blockchain/account/#treasury-account)
-  is defined at the genesis time and holds the total supply of the Pactus blockchain,
-  21 million coins. Each coin is divided into 1 billion units.
+  holds  21 million coins at the genesis time. Each coin is divided into 1 billion units.
+- **Reserved Accounts**: Reserved account are defined at the Genesis time and at total they have 21 milion coins.
+  These coins are reserved and the main purpise is to support the project over time.
```

Comment:
We should add a decription of the type of security for the reserve accounts.

Reply:
no doubt we will be asked questions if we dont put anything.

Reply:
Agreed! The best approach is to define a threshold signature address for these accounts (or, as YK once suggested, a vesting account).
A vesting account is an interesting idea; we can schedule the release of assets in the future.
However, the destination address is still a question mark.

We can even think about validators get approve for the signature (like 50% of total stake). Then here validators get too much influence on the network. 

At the moment, I think threshold signature is the best solution. Hopefully, we can implement it in the core.